scripts/metrics/metrics_xml_to_csv.py

```diff
@@ -0,0 +1,187 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Scrapes metrics from xml files and creates csv formatted files.
```

Comment:
Notably scrapes from a CTest XML, and a different scraper will need to be used for extracting metrics from other XML files. Should probably also change the file name to ctest_metrics_xml[...]